js/spreadsheet.js

```diff
@@ -1111,6 +1120,32 @@ function undoRedoToggle() {
   $('[data-action="redo"]').prop('disabled', disableRedo);
 }
 
+function undoRedo(event) {
```

Comment:
any specific reason you moved this in here?

Reply:
This file already contained some logic and functions related to other undo/redo functions so it was either move them all the the `undo-redo` file, or start adding those function names as global variable. It's all quite messy so I just put them all in the same file.